resources/assets/less/bem/comment.less

```diff
@@ -243,6 +243,10 @@
     .link-hover({
       color: @osu-colour-l1;
     });
+
+    @media @mobile {
+      padding: 5px 0;
```

Comment:
can use some margin for desktop as well as currently it's the only row with missing margin